gobblin-runtime/src/test/java/org/apache/gobblin/runtime/spec_catalog/FlowCatalogTest.java

```diff
@@ -161,6 +161,7 @@ public void createFlowSpec() {
       logger.info("[After Create] Spec " + i++ + ": " + gson.toJson(flowSpec));
     }
     Assert.assertTrue(specs.size() == 1, "Spec store should contain 1 Spec after addition");
```

Comment:
nit: for code quality should we mirror these assertions to use either assertEquals or assertTrue? I prefer the former (the way you did it)